pkg/term/term_windows.go

```diff
@@ -1,12 +1,15 @@
 // +build windows
+
 package term
 
 import (
+	"fmt"
 	"io"
 	"os"
+	"os/signal"
 
-	"github.com/Sirupsen/logrus"
-	"github.com/docker/docker/pkg/term/winconsole"
+	// "github.com/Sirupsen/logrus"
```

Comment:
In almost all of the rest of Docker, logrus has an alias 'log' so that calls are made to log.Debugx. This should be consistent even if not used so far.